tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 The MobileCoin Foundation
+
+from pathlib import Path
+import pytest
+
+
+@pytest.fixture
+def template_root():
```

Comment:
because of pytest fixtures and the tree like look up they use I moved the test expecteds to a parallel directory like structure.